surya.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LOGIN PAGE</title>
+</head>
+<body>
+    <form>
+        <div class="signin">
```

Comment:
why to put name signin in this place?

Reply:
signin and login, this two name are same so i give the name for sign in